src/screens/OtherProfile/index.js

```diff
@@ -164,13 +158,6 @@ const OtherProfile = () => {
   }, []);
 
   React.useEffect(() => {
-    netInfo.addEventListener((state) => {
-      if (state.isConnected) {
-        initData();
-      } else {
-        handleOfflineMode();
-      }
-    });
     initData();
   }, []);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The network connectivity check before calling `initData()` has been removed. Make sure that the app can handle situations when there's no network connection available. If `initData()` makes any network requests, they might fail without proper error handling.



<!-- This is an auto-generated comment by OSS CodeRabbit -->